lms/djangoapps/instructor_task/api.py

```diff
@@ -337,7 +337,7 @@ def submit_calculate_students_features_csv(request, course_key, features):
     """
     task_type = 'profile_info_csv'
     task_class = calculate_students_features_csv
-    task_input = {'features': features}
+    task_input = features
```

Comment:
@douglashall I would like to see at least a comment with a warning in front of the list of features that lets people know that adding new ones will potentially break microsites student reports, just so that we can avoid this issue coming up the next time someone wants to add something.